pkg/gui/controllers/stash_controller.go

```diff
@@ -165,6 +165,7 @@ func (self *StashController) handleRenameStashEntry(stashEntry *models.StashEntr
 				return err
 			}
 			self.context().SetSelectedLineIdx(0) // Select the renamed stash
```

Comment:
We call `.SetSelectedLineIdx(0)` in lots of places. Was this line changed for the sake of fixing a test? If so we ought to also be focusing the line elsewhere when we call `.SetSelectedLineIdx(0)`

Reply:
I'm happy with that